Age-ify.js

```diff
@@ -0,0 +1,5 @@
+console.log("Homework-1");
+let yearOfBirth = 1990;
+let yearFuture = 2045;
+let age =  yearFuture-yearOfBirth;
```

Comment:
Use `const` until it doesn't `let` you to.
`let` is for variables that are expected to be changed and reassigned further in the code.